core/db/migrate/20150515211137_fix_adjustment_order_id.rb

```diff
@@ -0,0 +1,66 @@
+class FixAdjustmentOrderId < ActiveRecord::Migration
+  def change
+    say 'Populate order_id from adjustable_id where appropriate'
+    execute(<<-SQL.squish)
```

Comment:
Yes because especially https://github.com/solidusio/solidus/commit/30ae28f2e24001214737130835a50d8f9fb7cdca only fixes data coming into the system. But will not fix missing data already in the system. And the data fix in https://github.com/solidusio/solidus/commit/b8b5e55661e2560437fb3b322ce284283e243dd8 is a point in time.

Between a spree/solidus user applied the earlier migrations and the moment this one (with a schema improvement) gets applied new bad data might have entered the system, via an unknown vector, or via the system running the old code while the new one is not deployed yet.

I'd not do any assumption in what state the DB is and always be idempotent when possible.

At least in our case we needed to back-fill `order_id` values, before creating the FK. Our system has massive amount of requests so this time based problems arise.